examples/react-native-bare-example/metro.config.js

```diff
@@ -0,0 +1,34 @@
+const {getDefaultConfig} = require('@react-native/metro-config');
+
+const path = require('path');
+
+// Find the project and workspace directories
+const projectRoot = __dirname;
+const workspaceRoot = path.resolve(projectRoot, '../..');
```

Comment:
If we have this, then this is referencing the local builds rather than the npm ones no?

Reply:
For this example and the expo one here, we shouldn't add this as it should be as close to a standalone as possible

Reply:
discussed offline and we'll keep this here: reason being is that because we are using workspaces and this is part of the workspace, the deps get hoisted, we do want to leverage as much of the hoisted deps as we can, so we need to tell metro where those things live.